config/qemu-keystone.toml

```diff
@@ -16,3 +16,19 @@ enable = false
 
 [policy]
 name = "keystone"
+
+[target.miralis]
+profile = "dev"
+start_address = 0x80000000
+stack_size = 0x8000
+
+[target.firmware]
+name="opensbi-jump"
+profile = "dev"
+start_address = 0x80200000
+
+[target.payload]
+name = "./images/Image"
```

Comment:
I suppose this is an internal path? We have a concept of artifacts that we can use for that. You can learn more about it [here](https://miralis-firmware.github.io/docs/tooling#test-artifacts) and see the existing list [here](https://miralis-firmware.github.io/docs/artifacts).

Eventually we will want to setup a repository to build all the required artifacts for Keystone. You can see an example [here]. The idea is simply to build every binary in the CI and expose them in the release, then we can update the [`artifacts.toml`](https://github.com/CharlyCst/miralis/blob/8d767b6a3f1605d33d0a38d545b431720fc3c221/misc/artifacts.toml) file to point to the new binaries.

If you feel stuck on porting Keystone to Miralis you can also look into that eventually :)

Reply:
Okay I see, should I make a pull request on [miralis-artifact-opensbi](https://github.com/CharlyCst/miralis-artifact-opensbi) for that?

Reply:
I removed the whole section `[target.payload]` since it will not be used anymore. 
I started using an OpenSBI+Keystone bundled as one firmware instead of loading `opensbi-jump` and the keystone payload separately.